popup.js

```diff
@@ -0,0 +1,202 @@
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://gabrielcoder247.github.io/',
+    source: 'https://github.com/gabrielcoder247/gabrielcoder247.github.io',
+  },
+  work3: {
+    id: 3,
+    name: 'Project name goes here...',
+    image: 'images/group-105.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://gabrielcoder247.github.io/',
+    source: 'https://github.com/gabrielcoder247/gabrielcoder247.github.io',
+  },
+  work4: {
+    id: 4,
+    name: 'Project name goes here...',
+    image: 'images/group-105.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://gabrielcoder247.github.io/',
+    source: 'https://github.com/gabrielcoder247/gabrielcoder247.github.io',
+  },
+  work5: {
+    id: 5,
+    name: 'Project name goes here...',
+    image: 'images/group-105.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://gabrielcoder247.github.io/',
+    source: 'https://github.com/gabrielcoder247/gabrielcoder247.github.io',
+  },
+  work6: {
+    id: 6,
+    name: 'Project name goes here...',
+    image: 'images/group-105.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://gabrielcoder247.github.io/',
+    source: 'https://github.com/gabrielcoder247/gabrielcoder247.github.io',
+  },
+};
+
+const projectData = [{
+  id: 1,
+  name: 'Project name goes here...',
+  image: 'images/laptop.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 2,
+  name: 'Project name goes here...',
+  image: 'images/screen.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 3,
+  name: 'Project name goes here...',
+  image: 'images/Rectangle-21devices.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
```

Comment:
- [ ] You are currently having your projects in `projectData` and `projectObj`. It is better to stick to the array of objects i.e. `projectData` to store your project details. It should be from this that you populate all your projects and the modal popup in the HTML file to display your projects. You are using `projectData` for your projects and `projectObj` for part of the modal popup